cookbooks/fb_systemd/recipes/default_packages.rb

```diff
@@ -59,6 +60,9 @@
        !['trusty', 'jessie'].include?(node['lsb']['codename'])
       systemd_packages << 'systemd-journal-remote'
     end
+    if node['fb_systemd']['timesyncd']['enable']
```

Comment:
Oh you're right, sorry I didn't expand out the diff. This is totally good.

Reply:
thanks!